torch_glow/tests/functionality/quantized_cut_in_the_middle.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import torch
+
+from tests.utils import GLOW_NODE_NAME, SUBGRAPH_ATTR
+import torch_glow
+import unittest
+
+
+class TestBlackList(unittest.TestCase):
+    def test_quantized_cut(self):
+        """Test cut quantized chunk in the middle."""
+        torch._C._jit_set_profiling_executor(False)
```

Comment:
do we need these btw?

Reply:
I remember last year if we dont add this there will be problems. I am not sure about the current status. Let me try.